src/metabase/lib/drill_thru.cljc

```diff
@@ -55,9 +61,35 @@
           (assoc-in context [:column :lib/source-uuid] (lib.options/uuid ag))))
       context))
 
-(defmethod lib.metadata.calculation/display-info-method ::drill-thru
-  [query stage-number drill-thru]
-  (lib.drill-thru.common/drill-thru-info-method query stage-number drill-thru))
+;;; TODO: Missing drills: automatic insights, format.
+(def ^:private available-drill-thru-fns
+  [#'lib.drill-thru.distribution/distribution-drill
+   #'lib.drill-thru.column-filter/column-filter-drill
+   #'lib.drill-thru.foreign-key/foreign-key-drill
+   #'lib.drill-thru.object-details/object-detail-drill
+   #'lib.drill-thru.pivot/pivot-drill
+   #'lib.drill-thru.quick-filter/quick-filter-drill
+   #'lib.drill-thru.sort/sort-drill
+   #'lib.drill-thru.summarize-column/summarize-column-drill
+   #'lib.drill-thru.summarize-column-by-time/summarize-column-by-time-drill
+   #'lib.drill-thru.underlying-records/underlying-records-drill
+   #'lib.drill-thru.zoom-in-timeseries/zoom-in-timeseries-drill])
+
+(mu/defn ^:private all-contexts :- [:sequential {:min 1} ::lib.schema.drill-thru/context]
+  [query                             :- ::lib.schema/query
+   stage-number                      :- :int
+   {:keys [dimensions], :as context} :- ::lib.schema.drill-thru/context]
+  (cons
+   context
```

Comment:
I'm still not really clear on whether the presence of `:dimensions` means we should ignore `:column` or not, I haven't really gotten a clear answer to this yet.